### PR TITLE
[exporter/tanzuobservability] update deprecated methods

### LINKED ISSUE #11508: lint failure
After the latest update of dependencies:

```
golangci-lint run --allow-parallel-runners
Error: exporter.go:87:12: SA1019: senders.NewProxySender is deprecated: Use 'senders.NewSender(url)' (staticcheck)
	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
	          ^
Error: metrics_exporter.go:32:12: SA1019: senders.NewProxySender is deprecated: Use 'senders.NewSender(url)' (staticcheck)
	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
	          ^
```

### PR DESCRIPTION
**Pull request by codeboten**
Fixes https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11508

### CODE REVIEW

**Comment by codeboten** [comment]
@keep94 please review to ensure this change makes sense. I wasn't sure what the right thing to do w/ `DistributionPort` is: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11510/files#diff-954a39c2d344b8fc3ad483d59a1a9a16c462ed6c8a7d14a580c49c5d5d3207cfL35

**Review by djaglowski — APPROVED** [review]
This looks correct to me. Approving, but waiting on code owner's review.

**Comment by djaglowski** [comment]
Since this addresses a problem that is blocking other PRs, I'm going to merge it now. 

@oppegard @thepeterstone @keep94 please review when you have a chance and open another PR if necessary.

**Review by keep94 — COMMENTED** [review]
I have a couple of minor comments. Otherwise looks good. I'd also like oppegard to have a look as he had a hand in this code as well.

File: exporter/tanzuobservabilityexporter/exporter.go [hunk 1]
```diff
@@ -84,13 +84,11 @@ func newTracesExporter(settings component.ExporterCreateSettings, c config.Expor
...
-	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
-		Host:                 tracingHost,
-		MetricsPort:          metricsPort,
-		TracingPort:          tracingPort,
-		FlushIntervalSeconds: 1,
-		SDKMetricsTags:       map[string]string{"otel.traces.collector_version": settings.BuildInfo.Version},
-	})
+	s, err := senders.NewSender(cfg.Traces.Endpoint,
+		senders.MetricsPort(metricsPort),
+		senders.FlushIntervalSeconds(1),
```
[the remark this answers is not in the document]
Comment (codeboten) [reply exporter/tanzuobservabilityexporter/exporter.go:89]: Thanks @keep94, please review https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11524

**Pull request merged**